corehq/warehouse/models/dimensions.py

```diff
@@ -36,8 +86,19 @@ class GroupDim(BaseDim):
 
     group_last_modified = models.DateTimeField()
 
+    @classmethod
+    def dependencies(cls):
+        return [GROUP_STAGING_SLUG]
```

Comment:
probably a good idea. not sure where we'd do it - maybe if we had a test that breaks the build if it finds a circular dependency?